frontend/src/metabase/query_builder/actions/core/parameterUtils.ts

```diff
@@ -29,33 +32,29 @@ type QueryParams = BlankQueryOptions & {
   objectId?: string;
 };
 
-function checkShouldPropagateDashboardParameters({
+function shouldPropagateDashboardParameters({
   cardId,
   deserializedCard,
   originalCard,
 }: {
   cardId?: number;
-  deserializedCard?: Card;
+  deserializedCard: Card;
   originalCard?: Card;
-}) {
-  if (!deserializedCard) {
-    return false;
-  }
+}): boolean {
+  // Either the card is saved and deserializedCard has parameters
```

Comment:
I _think_ this comment repeats what the code itself can communicate, i.e. seems to be more like a [_how_ comment rather than _a why_ comment](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/)

Reply:
Good point, I'll delete these